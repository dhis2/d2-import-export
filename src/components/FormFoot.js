import React from 'react'
import styles from './FormFoot.module.css'

export const FormFoot = ({ children }) => (
    <div className={styles.container}>{children}</div>
)
