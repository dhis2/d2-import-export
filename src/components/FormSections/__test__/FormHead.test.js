import React from 'react'
import renderer from 'react-test-renderer'
import { FormFoot } from '../FormFoot'

describe('Input component - FormFoot', () => {
    it('should render correctly', () => {
        const file = renderer
            .create(
                <FormFoot>
                    <span>Test</span>
                    Foobar
                </FormFoot>
            )
            .toJSON()

        expect(file).toMatchSnapshot()
    })
})
