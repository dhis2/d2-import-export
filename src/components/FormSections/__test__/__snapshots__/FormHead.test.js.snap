// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component - FormFoot should render correctly 1`] = `
<div
  className="container"
>
  <span>
    Test
  </span>
  Foobar
</div>
`;
